src/main/java/org/opentripplanner/updater/vehicle_parking/VehicleParkingDataSourceFactory.java

```diff
@@ -9,9 +9,9 @@ private VehicleParkingDataSourceFactory() {}
 
   public static DataSource<VehicleParking> create(VehicleParkingUpdaterParameters source) {
     switch (source.getSourceType()) {
-      case KML:           return new KmlBikeParkDataSource(source.getUrl(), source.getFeedId(), source.getNamePrefix(), source.isZip());
-      case PARK_API:      return new ParkAPIUpdater(source.getUrl(), source.getFeedId());
-      case OPEN_BIKE_BOX: return new OpenBikeBoxBikeRentalDataSource(source.getUrl(), source.getFeedId());
+      case KML:               return new KmlBikeParkDataSource(source.getUrl(), source.getFeedId(), source.getNamePrefix(), source.isZip());
+      case PARK_API:          return new CarParkAPIUpdater(source.getUrl(), source.getFeedId(), source.getTags());
+      case BICYCLE_PARK_API:  return new BicycleParkAPIUpdater(source.getUrl(), source.getFeedId(), source.getTags());
```

Comment:
Ah so OBB outputs ParkApi?

Reply:
Yes, but:
> :thinking: I'm not sure if the OpenBikeBox ParkAPI correctly differentiates between the cargo bike (rental) and the lockers (parking) -- that might need clarification.

```json
{
      "type": "vehicle-parking",
      "sourceType": "bicycle-park-api",
      "frequencySec": 30,
      "feedId": "OBB",
      "url": "https://backend.openbikebox.next-site.de/api/park-api/1.0/2",
      "tags": ["bike-locker"]
}
```

Reply:
So cargo bikes are retrieved as GBFS as they are rentals but the e-bike lockers should be retrieved via the OBB API since these are parking. Both use OBB as the backend however which is confusing.

Reply:
However, now that I think about it it's probably fine since ParkAPI _is_ a parking API, even though it's meant for cars.